examples/embassy-http-server/src/main.rs

```diff
@@ -101,7 +100,9 @@ fn main(spawner: Spawner, peripherals: pins::Peripherals) {
             button4: Input::new(buttons.btn4.degrade(), Pull::Up),
         };
 
-        ButtonInputs(make_static!(Mutex::new(buttons)))
+        static BUTTON_INPUTS: StaticCell<Mutex<CriticalSectionRawMutex, Buttons>> =
```

Comment:
I think `buttons` is not.

Reply:
```
error[E0435]: attempt to use a non-constant value in a constant
   --> examples/embassy-http-server/src/main.rs:104:45
    |
103 |         static BUTTON_INPUTS: ConstStaticCell<Mutex<CriticalSectionRawMutex, Buttons>> =
    |         -------------------- help: consider using `let` instead of `static`: `let BUTTON_INPUTS`
104 |             ConstStaticCell::new(Mutex::new(buttons));
    |                                             ^^^^^^^ non-constant value